make-all.sh

```diff
@@ -296,6 +291,6 @@ fi
 
 # Now build the rest of the recipe.
 config_recipe
-if [ ${_CFG_ONLY} -ne 0 ]; then
+if [ ${_CFG_ONLY} -eq 0 ]; then
```

Comment:
not equals 0?

Reply:
`_CFG_ONLY` is 0 if we want to configure and build, and non-zero if we want to configure without building.
I got the sense wrong when I originally coded it.